ibctest/relayer_factory.go

```diff
@@ -0,0 +1,49 @@
+package ibctest
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/ory/dockertest/v3"
+	"github.com/strangelove-ventures/ibctest/ibc"
+	"github.com/strangelove-ventures/ibctest/label"
+	ibctestrelayer "github.com/strangelove-ventures/ibctest/relayer"
+	"go.uber.org/zap/zaptest"
+)
+
+// RelayerFactory implements the ibctest RelayerFactory interface.
+type RelayerFactory struct{}
+
+// Build returns a relayer interface
+func (RelayerFactory) Build(
+	t *testing.T,
+	pool *dockertest.Pool,
```

Comment:
The most recent commit on ibctest (https://github.com/strangelove-ventures/ibctest/commit/9eab603b529ff8ce2a230985f5ae7db05cfab1a6) switched away from dockertest. I think if you pull that in, this just changes to `_ *client.Client` (because the `Relayer` type actually doesn't even use the value).

Reply:
updated